{"additions": 4, "auther_ref": "CMSSW_13_2_X", "auther_sha": "a4173651c33b0258447acbb332cfbf6ee5c2e539", "author": "alikaanguven", "body": "PR description:\r\nAddressing the issue:\r\nhttps://github.com/cms-sw/cmssw/issues/47777\r\n\r\nPR validation:\r\nStill python3. It should work.\r\n\r\nOrigin of backport:\r\nhttps://github.com/cms-sw/cmssw/pull/44201", "branch": "CMSSW_13_2_X", "changed_files": 1, "closed_at": "1744272940", "comments": 9, "commits": 1, "created_at": "1743694005", "deletions": 4, "labels": ["core-approved", "fully-signed", "tests-approved", "orp-approved", "backport-ok"], "merge_commit_sha": "23a039a5f0fde9af13cdf788d4861bb204c43729", "merged_at": "1744272940", "merged_by": "cmsbuild", "milestone": "CMSSW_13_2_X", "number": 47786, "release-notes": [], "review_comments": 0, "state": "closed", "title": "Backport LumiList.py bugfix to CMSSW_13_2_X", "updated_at": "1744272942", "user": "alikaanguven"}